### ISSUE TITLE
[sklearn#34651] ⚠️ CI failed on Unit tests Linux x86-64 pylatest_free_threaded (last failure: Aug 05, 2026) ⚠️

### ISSUE THREAD

**Issue by godofecht** [flooooooooooow/flow#300]
**Mirrored from [scikit-learn#34651](https://github.com/scikit-learn/scikit-learn/issues/34651)**

Original title: ⚠️ CI failed on Unit tests Linux x86-64 pylatest_free_threaded (last failure: Aug 05, 2026) ⚠️
Original labels: Needs Triage
Created: 2026-08-05T04:08:46Z

---

> [!WARNING]
> This issue is not yet ready for a PR. If you are interested in contributing to scikit-learn, please have a look at our [contributing guidelines](https://scikit-learn.org/dev/developers/contributing.html), and in particular the sections for [new contributors](https://scikit-learn.org/dev/developers/contributing.html#new-contributors) and the ["Needs triage"](https://scikit-learn.org/dev/developers/contributing.html#issues-tagged-needs-triage) label.

**CI failed on [Unit tests Linux x86-64 pylatest_free_threaded](https://github.com/scikit-learn/scikit-learn/actions/runs/30972803093/job/92200719120)** (Aug 05, 2026)
- test_filter_warning_propagates_no_side_effect_with_loky_backend

---

## Flow context

This issue is mirrored from scikit-learn to explore whether Flow encounters the same problem or can approach it differently. Add analysis below.

**Comment by godofecht** [comment]
Moved to https://github.com/godofecht/flow-scikit/issues/123 in the flow-scikit repository. Further library-specific triage and implementation will happen there.
